erpnext/accounts/doctype/pricing_rule/pricing_rule.json

```diff
@@ -12,6 +13,7 @@
   "apply_on",
   "price_or_product_discount",
   "warehouse",
+  "condition",
```

Comment:
should in a different section

Reply:
Moved to Added in "Advanced Settings" section.
Is that fine?